applications/luci-app-ddns/po/templates/ddns.pot

```diff
@@ -1,5 +1,15 @@
 msgid ""
-msgstr "Content-Type: text/plain; charset=UTF-8"
+msgstr ""
```

Comment:
I am not sure if it worthwhile to add the comment block there. The language syncronising script (build/i18n-sync.sh that runs i18n-scan.pl and i18n-update.pl ) will probably clean it away the next time when that script is run. That happened at least the last time: https://github.com/openwrt/luci/commit/8ae57f62044ab9ac73a9113417e33c23aedc1974 
I am not sure if it does not like comment headers in the template .pot files, although it allows it for the actual language .po files).